api.md

```diff
@@ -97,6 +103,43 @@ Response
 }
 ```
 
+
+#### `DELETE /api/v2/device/:deviceID` - deletes the device from your tailnet
```

Comment:
From which tailnet?

Or is an API key bound to a specific tailnet that's implicit? But if so, why do the other APIs require a tailnet ID?